### DISCUSSION TITLE
how to extend models?

### DISCUSSION THREAD

**Discussion by dcsan** [prisma/prisma#11951] [Q&A] [answered]
with prisma how can i extend a model to add my own methods?

even mongoose has instance methods as one of the basic concepts
https://mongoosejs.com/docs/guide.html#methods

https://www.prisma.io/docs/concepts/components/prisma-client/custom-models

Really basic stuff, surprised prisma for all its magic doesn't support- even prisma 2

https://github.com/prisma/prisma/issues/5998

**Comment by austincrim** [comment, marked as answer]
Hey @dcsan 👋 ,

We have a few outstanding feature requests related to your question:
#7161
#5998
#5273

Please feel free to add a +1 or your specific use case to any of those issues if you feel it hasn't been covered already. This really helps our product and engineering teams prioritize!

Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
